docs_user/modules/proc_deploying-file-systems-service-control-plane.adoc

```diff
@@ -1,181 +1,43 @@
-* If you had multiple storage backend drivers configured with {OpenStackShort} {rhos_prev_ver},
-you will need to split them up when deploying {rhos_acro} {rhos_curr_ver}. Each storage
-backend driver needs to use its own instance of the `manila-share`
-service.
-* If a storage backend driver needs a custom container image, find it on the
-https://catalog.redhat.com/software/containers/search?gs&q=manila[RHOSP Ecosystem Catalog]
-and create or modify an `OpenStackVersion` manifest to specify the custom image
-using the same `<custom name>`.
-+
-In the following example that illustrates multiple storage backend drivers,
-where only one is using a custom container image, we have the manila spec from
-`OpenStackControlPlane`:
-+
-[source,yaml]
-----
-  spec:
-    manila:
-      enabled: true
-      template:
-        manilaAPI:
-          customServiceConfig: |
-            [DEFAULT]
-            enabled_share_protocols = nfs
-          replicas: 3
-        manilaScheduler:
-          replicas: 3
-        manilaShares:
-         netapp:
-           customServiceConfig: |
-             [DEFAULT]
-             debug = true
-             enabled_share_backends = netapp
-             host = hostgroup
-             [netapp]
-             driver_handles_share_servers = False
-             share_backend_name = netapp
-             share_driver = manila.share.drivers.netapp.common.NetAppDriver
-             netapp_storage_family = ontap_cluster
-             netapp_transport_type = http
-           replicas: 1
-         pure:
-            customServiceConfig: |
-             [DEFAULT]
-             debug = true
-             enabled_share_backends=pure-1
-             host = hostgroup
-             [pure-1]
-             driver_handles_share_servers = False
-             share_backend_name = pure-1
-             share_driver = manila.share.drivers.purestorage.flashblade.FlashBladeShareDriver
-             flashblade_mgmt_vip = 203.0.113.15
-             flashblade_data_vip = 203.0.10.14
-            replicas: 1
-----
-+
-And then we have the `OpenStackVersion` to define the custom container image:
-+
-[source,yaml]
-----
-apiVersion: core.openstack.org/v1beta1
-kind: OpenStackVersion
-metadata:
-  name: openstack
-spec:
-  customContainerImages:
-    cinderVolumeImages:
-      pure: registry.connect.redhat.com/purestorage/openstack-manila-share-pure-rhosp-18-0
-----
-+
-The name of the `OpenStackVersion` must match the name of your `OpenStackControlPlane`, so in your case it may be other than `openstack`.
-
-. If providing sensitive information, such as passwords, hostnames and
-usernames, it is recommended to use {OpenShift} secrets, and the
-`customServiceConfigSecrets` key. An example:
+. Optional: If you are providing sensitive information, such as passwords, hostnames, and usernames, create a secret CR and apply it to the relevant {rhos_component_storage_file}.
+.. Create a file that includes the secrets, for example, `netapp_secrets.conf`:
```

Comment:
@fmount or @gouthampacha I separated the step about secrets into multiple steps. Is the wording and the flow accurate?

Reply:
I think that module looks good so far, and I'm ok to move steps 3a and 3b about creating `netapp` secrets to that section, so people know that, as a prereq, if they're using a 3rd party storage that requires credentials, they need to create that secret that will be referenced in the manila CR/patch file via  `customServiceConfigSecrets` interface.